Integrations/SplunkPy/SplunkPy.py

```diff
@@ -25,7 +25,7 @@
 FETCH_LIMIT = int(demisto.params().get('fetch_limit', 50))
 FETCH_LIMIT = max(min(200, FETCH_LIMIT), 1)
 PROBLEMATIC_CHARACTERS = ['.']
-REPLACE_WITH = '_'
+REPLACE_WITH = '_' if True else False
```

Comment:
?
this should be determine with an integration parameter: `should replace dots in incident fields`